src/core/evaluator.js

```diff
@@ -783,6 +783,7 @@ class PartialEvaluator {
       })
       .then(translated => {
         state.font = translated.font;
+        state.fontSize = fontArgs[1];
```

Comment:
The `fontArgs` isn't guaranteed to be defined, thus causing this to throw in some cases, please define `fontSize` similar to the existing `fontName` handling further up in this method.